Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -1627,7 +1685,362 @@ script:
+      description: Username
+      type: string
+    - contextPath: PaloAltoNetworksXDR.EndpointViolations.violations.ip
+      description: 'IP address '
+      type: string
+    - contextPath: PaloAltoNetworksXDR.EndpointViolations.violations.timestamp
+      description: Timestamp of the violation
+      type: number
+    - contextPath: PaloAltoNetworksXDR.EndpointViolations.violations.violation_id
+      description: Violation ID
+      type: number
+    - contextPath: PaloAltoNetworksXDR.EndpointViolations.violations.type
+      description: Type of violation
+      type: string
+    - contextPath: PaloAltoNetworksXDR.EndpointViolations.violations.vendor_id
+      description: Vendor ID
+      type: string
+    - contextPath: PaloAltoNetworksXDR.EndpointViolations.violations.vendor
+      description: Name of vendor
+      type: string
+    - contextPath: PaloAltoNetworksXDR.EndpointViolations.violations.product_id
+      description: Product ID
+      type: string
+    - contextPath: PaloAltoNetworksXDR.EndpointViolations.violations.product
+      description: Name of product
+      type: string
+    - contextPath: PaloAltoNetworksXDR.EndpointViolations.violations.serial
+      description: Serial Number
+      type: string
+    - contextPath: PaloAltoNetworksXDR.EndpointViolations.violations.endpoint_id
+      description: Endpoint ID
+      type: string
+  - arguments:
+    - description: Comma separated list of endpoint IDs
+      isArray: true
+      name: endpoint_ids
+      required: true
+    - description: 'The type of platform and files you want to retrieve - comma separated
+        list of strings. '
+      isArray: true
+      name: windows_file_paths
+    - description: 'The type of platform and files you want to retrieve - comma separated
+        list of strings. '
+      isArray: true
+      name: linux_file_paths
+    - description: 'The type of platform and files you want to retrieve - comma separated
+        list of strings. '
+      isArray: true
+      name: mac_file_paths
+    description: Retrieve files from selected endpoints. You can retrieve up to 20
+      files, from no more than 10 endpoints. At least one endpoint ID and one file
+      path are necessary in order to run the command. After running this command,
+      you can use - xdr-action-status-get command with returned action_id, to check
+      action status.
+    name: xdr-retrieve-files
+    outputs:
+    - contextPath: PaloAltoNetworksXDR.RetrievedFiles.action_id
+      description: ID of action to retrieve files from selected endpoints.
+      type: string
+  - arguments:
+    - description: Action ID retrieved from the retrieve file command.
+      isArray: true
+      name: action_id
+      required: true
+    - auto: PREDEFINED
+      default: true
+      defaultValue: "true"
+      description: Choose whether you want to attach retrieved files to War room or
+        not. Default value - true.
```

Comment:
```suggestion
      description: If set to true the retrieved files will be saved in the war room otherwise a link to download the file 
      will be retrieved. Default value - true.
```